### PR TITLE
[MU4] Sync CI scripts from 3.x to master

### PR DESCRIPTION
**Pull request by igorkorsukov**
Sync CI scripts from 3.x to master

### CODE REVIEW

File: build/ci/linux/runmtests.sh [hunk 1]
```diff
@@ -7,17 +7,18 @@ df -k .
...
 make -j2
 
 df -k .
 
-xvfb-run -a ctest -j2 --output-on-failure
+ctest -j2 --output-on-failure
 
-df -k .
\ No newline at end of file
+df -k .
```
Comment (Jojo-Schmitz) [inline build/ci/linux/runmtests.sh:24]: Not quite sure what the purpose of these is (I guess some debugging output), but a) IIRC `-k` is the default as per POSIX, and b) Linux' df's `-h' might be more helpful, 'human readable'?

Comment (igorkorsukov) [reply build/ci/linux/runmtests.sh:24]: We had a problem with free space.
I added this for monitoring
I will change to `-h`, thanks.

Comment (igorkorsukov) [reply build/ci/linux/runmtests.sh:24]: https://github.com/musescore/MuseScore/pull/6687

Comment (Jojo-Schmitz) [reply build/ci/linux/runmtests.sh:24]: Didn't notice those back then ;-)

**Pull request merged**